samples/bluetooth/direction_finding_connectionless_rx/src/main.c

```diff
@@ -0,0 +1,359 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stddef.h>
+#include <errno.h>
+#include <zephyr.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/direction.h>
+
+#define DEVICE_NAME CONFIG_BT_DEVICE_NAME
+#define DEVICE_NAME_LEN (sizeof(DEVICE_NAME) - 1)
+#define PEER_NAME_LEN_MAX 30
+#define TIMEOUT_SYNC_CREATE K_SECONDS(10)
+
+static struct bt_le_per_adv_sync_param sync_create_param;
```

Comment:
sync_create_params can be declared on the stack where it is used.

Reply:
Changed to local var in `create_sync`.